worldengine/cli/main.py

```diff
@@ -391,8 +391,8 @@ def main():
     if args.seed is not None:
         seed = int(args.seed)
     else:
-        seed = random.randint(0, 65535)#RNG initialization is done automatically
-    random.seed(seed)
+        seed = numpy.random.randint(0, 999999)  # RNG initialization is done automatically
```

Comment:
This could be undone, if it seems problematic. I didn't see a reason to restrict worldengine to 2^16 different outputs (when no seed is passed). I am not even sure if Python supports anything less than a 32 bit integer. (The reason for not choosing a higher value is the multiplication of the seed in generation.py.)